assets/components/atoms/select/select.scss

```diff
@@ -1 +1,139 @@
+
+.ms-parent {
+  display: block;
+  position: relative;
+}
+
+.ms-choice {
+  display: block;
+  width: 100%;
+  height: $custom-select-height;
+  padding: $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding) $custom-select-padding-y $custom-select-padding-x;
+  overflow: hidden;
+  white-space: nowrap;
+  background: $custom-select-bg $custom-select-indicator no-repeat right $custom-select-padding-x center;
+  background-size: $custom-select-bg-size;
+  border: $custom-select-border-width solid $custom-select-border-color;
+  border-radius: $custom-select-border-radius;
+  text-align: left;
+  text-decoration: none;
+  line-height: $custom-select-line-height;
+  color: $custom-select-color;
+
+  &:focus {
+    border-color: $custom-select-focus-border-color;
+    box-shadow: $custom-select-focus-box-shadow;
+    outline: none;
+  }
+
+  &.disabled {
+    background-color: $custom-select-disabled-bg;
+    background-image: none;
+    border: $custom-select-border-width solid $custom-select-border-color;
+    cursor: default;
+  }
+
+  > span {
+    display: block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+
+    &.placeholder {
+      color: $input-placeholder-color;
+    }
+  }
+
+  > div {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 20px;
+    height: 25px;
+    background: url('multiple-select.png') left top no-repeat;
+
+    &.open {
+      background: url('multiple-select.png') right top no-repeat;
+    }
+  }
+}
+
+.ms-drop {
+  display: none;
+  position: absolute;
+  width: 100%;
+  min-width: $dropdown-min-width;
+  overflow: hidden;
+  margin-top: -$dropdown-border-width;
+  padding: $dropdown-padding-y 0;
+  z-index: 1000;
+  background: $dropdown-bg;
+  border: $dropdown-border-width solid $dropdown-border-color;
+  color: $dropdown-link-color;
+  box-shadow: $dropdown-box-shadow;
+
+  &.bottom {
+    top: 100%;
+    border-bottom-right-radius: $dropdown-border-radius;
+    border-bottom-left-radius: $dropdown-border-radius;
+  }
+
+  &.top {
+    bottom: 100%;
+    border-top-right-radius: $dropdown-border-radius;
+    border-top-left-radius: $dropdown-border-radius;
+  }
+
+  ul {
+    overflow: auto;
+    margin: 0;
+    padding: 0;
+
+    > li {
+      list-style: none;
+      background-image: none;
+
+      .disabled {
+        color: $dropdown-link-disabled-color;
+      }
+
+      &.multiple {
+        display: block;
+        float: left;
+      }
+
+      &.group {
+        clear: both;
+      }
+    }
+  }
+
+  label {
+    display: block;
+    margin-bottom: 0;
+    padding: $dropdown-item-padding-y $dropdown-item-padding-x;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+
+    &:hover {
+      background-color: $dropdown-link-hover-bg;
+      color: $dropdown-link-hover-color;
+    }
+
+    &.optgroup {
+      font-weight: $font-weight-bold;
+    }
+  }
+
+  input[type="checkbox"] {
```

Comment:
Not possible to extend B4 custom checkbox for better consistency ?

Reply:
I would have loved that too, but the plugin doesn't allow to change its markup output. :/